src/bms.lisp

```diff
@@ -0,0 +1,93 @@
+(defun bms-loop () {
+    (var vmin-limit)
+    (var vmax-limit)
+    (var tmax-limit)
+    (var tmin-limit -10)
```

Comment:
Why did you silently change this to -10? When you change the semantics, could you **please** comment with an explanation?